documentation/preview/README.md

```diff
@@ -0,0 +1,86 @@
+# Azure PowerShell module 4 preview - User documentation
+
+This article explains how to get started with the preview of the Az 4.x PowerShell module.
+
+## Benefits
+
+With this new module we are introducting several changes to Azure PowerShell. The links below will provide more details for each feature and allow you to submit comments:
+
+- [Add support for Azure profiles](../RFC/RFC0001-Azure-Profiles.md)
```

Comment:
I think we want to point at the `Feature` documents I created here, and add a section on Feedback that points to the RFCs.

Reply:
That makes sense, I had this part written before we the `Feature` document exist.